roles/unicorn_user/tasks/main.yml

```diff
@@ -16,6 +16,17 @@
     state: present
   with_flattened: "{{ users_sysadmin }}"
   become: yes
+  when: ssh_key_path is undefined
+
+# Add ssh key path to run playbook command with --extra-vars
+# ex.: --extra-vars "ssh_key_path=.ssh/id_rsa.pub"
+- name: add ssh key alternative
+  authorized_key:
+    user: "{{ unicorn_user }}"
+    key: "{{ lookup('file', lookup('env','HOME') + ssh_key_path) }}"
```

Comment:
It would be a bit more obvious for the user if `ssh_key_path` behaved as an actual path allowing `ssh_key_path=~/.ssh/id_rsa.pub` and also more flexible; it would allow public keys from any location.

So I suggest this to be

```sh
key: "{{ ssh_key_path }}"
```

Reply:
I copied this from the other ssh key setup for the default user in https://github.com/openfoodfoundation/ofn-install/blob/530741c5d82f6c47b0e196666b0b13920310fedf/roles/default_user/tasks/main.yml#L20

Is it worth changing that one as well?

Reply:
It looks like they have to match for it to run, so I guess yes is the answer...

Reply:
On testing it, it looks like it won't accept paths that start with `~/`, which I guess is why the same method of using `{{ lookup('file', lookup('env','HOME') + ssh_key_path) }}` was used previously in /roles/default_user/tasks/main.yml.

Reply:
Ah ok, nevermind then... Although I find it really weird...

Reply:
The answer was somewhere in between: `{{ lookup('file', ssh_key_path) }}` can handle the tilde in the filepath, the previous addition of `lookup('env','HOME')` wasn't needed.